src/test/java/sg/ncl/ConnectionPropertiesTest.java

```diff
@@ -213,6 +214,7 @@ public void testSetTelemetryAddress() throws Exception {
         assertThat(properties.getTelemetryAddress()).isEqualTo(one);
     }
 
+    @Ignore
     @Test
     public void testGetTelemetryPort() throws Exception {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Fix or remove this skipped unit test [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS1607)